Source/Factory/Internal/NSInvocation+TCFWrapValues.h

```diff
@@ -17,4 +17,7 @@
 /** Get argument at index, wrapping primitive values in NSValue if needed */
 - (id)typhoon_getArgumentObjectAtIndex:(NSInteger)idx;
 
+/** Get return value, wrapping primitive values in NSValue if needed*/
+- (id)typhoon_getReturnValue;
```

Comment:
@alexgarbarev It used for case if we create factory definition for example from UIScreen, that has bounds (CGRect). Looks please https://github.com/appsquickly/Typhoon/pull/518 :)